setup.py

```diff
@@ -20,9 +20,10 @@
 
 
 from setuptools import find_packages, setup
+from marquez_client.version import VERSION
```

Comment:
I think for  `setup.py` it is recommended to actually read/parse the content of the file instead of importing it. (See [single-sourcing-package-version](https://packaging.python.org/guides/single-sourcing-package-version/))

Reply:
Fixed as part of the comment above. Please review